project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject ring/ring-ssl "0.2.1"
+(defproject ring/ring-ssl "0.2.2-SNAPSHOT"
```

Comment:
Please don't change the version number in PRs. A PR should contain just the code necessary to make the change.